src_c/polygon.c

```diff
@@ -341,7 +367,90 @@ pg_polygon_copy(pgPolygonObject *self, PyObject *_null)
     return pgPolygon_New(&self->polygon);
 }
 
+static PyObject *
+pg_polygon_move(pgPolygonObject *self, PyObject *const *args, Py_ssize_t nargs)
+{
+    double x = 0, y = 0;
+    switch (nargs) {
+        case 1:
+            if (PySequence_Check(args[0])) {
```

Comment:
you can directly call `pg_TwoDoublesFromObj` without the extra sequence check

Reply:
I think this one also is from move function, so I have changed that one as itzpr said me to do